src/examples/design/icons/SvgSearch.tsx

```diff
@@ -68,6 +68,7 @@ interface ISvgSearchProps {
   data: {
     edges: ISvgNodeProps[];
   };
+  collapsible: boolean;
```

Comment:
This prop configures the default uncontrolled state, so a more suitable name is `defaultCollapsed`. This follows the general conventions of HTML & React elements:

```jsx
const Example = (
  <>
    <input type="text" defaultValue="hello world" /> 
    <input type="checkbox" defaultChecked /> 
  </>
)
```

Reply:
Gotcha, I'll make the changes. 😁